contracts/tg4-stake/src/claim.rs

```diff
@@ -155,12 +155,13 @@ impl<'a> Claims<'a> {
     /// This iterates over all mature claims of any addresses, and removes them. Up to `limit`
     /// claims would be processed, starting from the oldest. It removes the finished claims and
     /// returns vector of pairs: `(addr, amount)`, representing amount of tokens to be released to particular addresses
+    #[allow(clippy::type_complexity)]
```

Comment:
I am wondering - this is starting to being not obvious to read. There is a comment about what those pairs are. Why not simplify this by creating a simple type alias:

```rust
type TokensReleaseInfo = (Addr, Uint128);
```
So the return type becomes:

```rust
StdResult<Vec<TokenReleaseInfo>, VecTokenReleaseInfo>>
```

I know it is used as a helper function, but I would even go a step further and make it into:

```rust
struct TokenReleaseInfo {
  addr: Addr,
  amount: Uint128,
}

struct ReleaseData {
  liquid_releases: TokenReleaseInfo,
  vesting_releases: TokenReleaseInfo,
}
```

As it is a bit more to write now, but it makes it far easier to read - and now I read this contract after half a year or so, and I am struggling to understand what is what here...

Not to mention, that comment was not updated and didn't match returned data which makes it even harder to reason about (by the way - that is why semantically named types are better than comments - you don't rely on it to be updated while compilation doesn't fail by them).

Reply:
Will do. And thanks for pointing out to outdated comment.